nbs/tutorials/07-forest_plot.ipynb

```diff
@@ -17,9 +17,9 @@
    "id": "cfdb7e31",
```

Comment:
<h3>Example for using reviewnb to modify the tutorial notebook.</h3><p> </p><p>The forest plot function is added later, so technically it's not supported neither with the updates of DABEST v2023.02.14 nor v2024.03.29, please consider a change or placeholder to remind the change of subsequent version number</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCLAB/DABEST-python/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I've changed it to a place holder like 20xx.xx.xx, is this scenario ok to include in the tutorial notebook?</p>